src/commands/fun/twitch.js

```diff
@@ -0,0 +1,62 @@
+const SA = require('superagent')
+const Command = require('../../classes/Command')
+
+module.exports = new Command(async (msg, suffix) => {
+	if (!suffix) return msg.channel.createMessage('No channel specified!')
+	const user = await getUser(suffix);
+	if(!user) return msg.channel.createMessage(`**${suffix}** isn't a valid channel.`)
+	SA.get(`https://api.twitch.tv/kraken/streams/${user}`)
+		.set({ 'Accept': 'application/vnd.twitchtv.v5+json', 'Client-ID': process.env.TWITCH_ID })
+		.end((err, response) => {
+			if (err) {
+				global.logger.error(err)
```

Comment:
```suggestion
				logger.error('REST', err)
```

Reply:
Identify that this error came from the twitch command